src/redux/error/error.reducer.js

```diff
@@ -0,0 +1,19 @@
+import { SET_ERROR } from './error.types';
+
+const initialState = {
+  error: null
```

Comment:
How does this error reducer influence `UI`?

Reply:
This error reducer set error object to the store and after that error-page can view to user type of error in different languages.